contrib/tele.centrum.v2/index.php

```diff
@@ -3,13 +3,22 @@
 error_reporting(E_ALL &~ E_NOTICE &~ E_DEPRECATED);
 
 require_once('..' . DIRECTORY_SEPARATOR . 'initLMS.php');
+require_once('lib' . DIRECTORY_SEPARATOR . 'definitions.php');
 
 $uid        = $_GET['id'];
 $phone      = $_GET['phone'];
 $agentnr    = $_GET['agentnr'];
 $ticket['phonetype'] = 'on';
 
-require_once('lib' . DIRECTORY_SEPARATOR . 'definitions.php');
+$basedir=(__DIR__ . DIRECTORY_SEPARATOR . 'templates_c');
+$wwwuser=posix_getuid();
+$wwwgroup=posix_getgid();
+
+if (!is_dir($basedir)) {
+    die("Please create Smarty compiled templates directory using shell command: mkdir " . $basedir);
+} elseif (!is_readable($basedir) || !is_writable($basedir)) {
+    die("Please set valid permissions to Smarty compiled templates directory using shell commands: chmod 755 " . $basedir . "; chown " . $wwwuser. ":" . $wwwgroup . " " . $basedir);
```

Comment:
valid? Wiesz co to oznacza?

Reply:
Niestety Google Translator nie radzi sobie za dobrze z kontekstami tłumaczeń i takie potem efekty...

Reply:
Czytający, która włada natywnie angielski zrozumie zapewne ten zwrot, ale to jest coś jak by zamiast dać:
```
Ustaw poprawne uprawnienia ...
```
dać
```
Ustaw dobre uprawnienia ...
```

Reply:
To moze propper?

Reply:
Aż musiałem to sprawdzić...
https://pl.bab.la/slownik/angielski-polski/valid

Są znaczenia: prawidłowy, odpowiedni, uzasadniony.

No nic zmieniłem na propper.